apps/src/p5lab/gamelab/GameLab.js

```diff
@@ -69,6 +73,26 @@ export default class GameLab extends P5Lab {
   }
 
   runButtonClick() {
+    // For AI Rubrics Pilot
```

Comment:
This comment was intentionally kept in here since this piece is only for the pilot at the moment.  I can see this being forgot about when we remove this feature from under the pilot AND teacher tools isn't in GameLab a lot - which is why I thought this comment might be needed for other folks who look at this code.